Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py

```diff
@@ -0,0 +1,5 @@
+import demistomock as demisto  # noqa: F401
```

Comment:
- please add **main** method
- don't use demisto.results() Use return_results() instead
- surround the code with **try** **except** block
- do the same for **CreatePrismaCloudComputeComplianceReportButton**